bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -120,6 +121,8 @@ for trigger_file, injection_file in zip(args.trigger_files,
                 ends = f['/segments/%s/end' % key][:]
                 any_seg += events.start_end_to_segments(starts,ends)
         ana_start, ana_end = events.segments_to_start_end(any_seg)
+        any_seg.coalesce()
+        foreground_time_exc = abs(any_seg)
```

Comment:
It's not clear to me whether this is the right calculation for 'foreground_time_exc' i.e. total analysis time excluding small windows around zerolag events.  If we don't know that this is right we need a 'FIXME' here saying so.